src/suites/positive/conductors/validation-conductor.ts

```diff
@@ -14,22 +15,25 @@ export default class ValidationConductor {
 
   private exampleGroup: ExampleGroup;
 
-  private requestBody: RequestBody;
+  private exampleRequestBody: ExampleRequestBody;
+
+  private operationExampleName: string;
 
   private securityValues: SecurityValues;
 
   private server: string | undefined;
 
   constructor(
     schema: OASSchema,
-    { operation, exampleGroup, requestBody }: OperationExample,
+    { operation, exampleGroup, exampleRequestBody, name }: OperationExample,
```

Comment:
OperationExample was changed to be created with three objects but here four are returned

Reply:
Assigned the `name` property to a variable called `operationExampleName`